product_state/data/product_state_data.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
```

Comment:
<odoo noupdate="1">

Reply:
But I set the noupdate=1 in data. Is it needed also in odoo?

Reply:
you write `noupdate=1` in odoo tag also,  like `<odoo noupdate=1>`
and now no need to <data> tag for reducing unused line